### DISCUSSION TITLE
I am not able to setup host for my website in my GitHub account in Pages

### DISCUSSION THREAD

**Discussion by Raki-Design** [community/community#121052] [New to GitHub] [answered]
I get an error 404 page when i click save in the domain name option after selecting the main branch.
I tryied to set it up with 3 domain names that are:

rakidesign.github.io
raki-design.github.io
raki-design.github.com

**Comment by gurjeetsinghvirdee** [comment, marked as answer]
Hey @Raki-Design 
https://github.com/Raki-Design/raki-design.github.io this repository is empty, there is no branches & code inside it 
You can't host a website without code

**Reply by Raki-Design** [reply]
It seems I was forgetting to hit Commit changes to save the files I uploaded. Thank you very much for your help mister!

**Reply by gurjeetsinghvirdee** [reply]
No worries, If you find it useful `mark it as answered` & don't upload your file instead push your changes through ide
watch this it will help you https://www.youtube.com/watch?v=i_23KUAEtUM&t 
Happy Coding!

**Comment by elyasasadi** [comment]
[``](url)
